File: example/src/net/SDK_V3/sample.vb
import Flagship = Flagship

Private Sub SurroundingSub()
    Dim client As [let] = FlagshipBuilder.Start("ENV_ID", "API_KEY")

    Dim context = New Dictionary(Of String, Object)()
    context.Add("key", "value")

    Dim visitor As [let] = client.NewVisitor("visitor_id", context)
    visitor.FetchFlags();

    Dim btnColorFlag As [let] = visitor.GetFlag("btnColor", 'red');
End Sub

File: example/src/net/SDK_V3/sample.fs
open System.Collections.Generic
open Flagship

let client = FlagshipBuilder.Start("ENV_ID","API_KEY");

let context = new Dictionary<string, obj>();
context.Add("key", "value");

let visitor = client.NewVisitor("visitor_id", context);

visitor.FetchFlags();
let btnColorFlag = visitor.GetFlag("btnColor", 'red');